src/_data/styles/modern-stars.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Modern stars",
+  "author": "Alba Maldonado",
+  "stylesheet": "https://raw.githubusercontent.com/alba02/stylestage-modern-stars/080fc9be245267c2609ffa8683a179cd870e8e75/modern-stars"
```

Comment:
Hi @alba02 - please save this file to include `.css` in your repository and then update the URL here as well! The Style Stage process isn't able to get the content since it's not recognizing it as a stylesheet.